track.py

```diff
@@ -364,11 +492,12 @@ def crop(dfIn):
 
         self.speedLimits = crop(self.speedLimits)
         self.gradients = crop(self.gradients)
+        self.curvatures = crop(self.curvatures)
 
 
 if __name__ == '__main__':
 
     # Example on how to load and plot a track
 
-    track = Track(config={'id':'00_stationX_stationY'})
+    track = Track(config={'id':'00_stationX_stationY', 'clothoidSamplingInterval': 10})
```

Comment:
why do you pass the clothoid sampling interval option if the track has no curvature information?